<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThingRuleEdit</class>
 <widget class="QDialog" name="ThingRuleEdit">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>101</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>574</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактирование правила</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Если</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="param"/>
     </item>
     <item>
      <widget class="QCheckBox" name="negative">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>не</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="operand">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>равно</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>содержит</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>больше</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>меньше</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="value_str">
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Действие:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="action">
       <item>
        <property name="text">
         <string>отображать</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>не отображать</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>следующее правило</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="itemColorLb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Цвет:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorButton" name="itemColorBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="OkCancelBtn">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>subclasses/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>param</tabstop>
  <tabstop>negative</tabstop>
  <tabstop>operand</tabstop>
  <tabstop>value_str</tabstop>
  <tabstop>action</tabstop>
  <tabstop>itemColorBtn</tabstop>
  <tabstop>OkCancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>OkCancelBtn</sender>
   <signal>rejected()</signal>
   <receiver>ThingRuleEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OkCancelBtn</sender>
   <signal>accepted()</signal>
   <receiver>ThingRuleEdit</receiver>
   <slot>okBtnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okBtnClick()</slot>
 </slots>
</ui>
